ansible/roles/matrix/templates/homeserver.yaml

```diff
@@ -29,7 +32,7 @@ listeners:
   # For when matrix traffic passes through loadbalancer that unwraps TLS.
   - port: 55156
```

Comment:
Since you are cleaning up this file: is `55156` the default port for matrix? If so, let's change it to a random port number. It's an anal way to make sure the config file is being applied.

Reply:
It's not the default port number.